UKHomeOffice/cop-ui
# COP-2831: View completed form in Case History

**Description**
This branch adds the functionality to display completed forms in the case history, along with tests. 

**To test:**
- Pull branch and run locally in development mode, making sure you are using the cop-dev proxy in the package.json.
- Search for a case with multiple form instances e.g. DEV-20210205-2681. Under CaseHistory click on the case to reveal the forms associated with the case in a list of drop downs links.
- Open one and click the "Show Details" button. This should display the form. Click Hide Details to remove snapshot or click into another form.
